documentation/api/io.strimzi.api.kafka.model.listener.NodePortListenerConfiguration.adoc

```diff
@@ -0,0 +1,30 @@
+Use `NodePortListenerConfiguration` properties to add configuration to external node port listeners.
+
+[id='property-nodeport-listener-preferredAddressType-{context}']
+.`preferredAddressType`
+
+Use the `preferredAddressType` property in your listener configuration to specify the first address type checked as the node address.
+This property is useful, for example, if your deployment does not have DNS support, or you only want to expose a broker internally through an internal DNS or IP address.
+If an address of this type is found, it is used.
+If the preferred address type is not found, Strimzi proceeds through the types in the standard order of priority.
```

Comment:
Can we link to where that standard order is defined?

Reply:
> Strimzi proceeds through the types in the xref:nodeport-listener-{context}[standard order of priority].